tests/unit/dojson/test_dojson_hep_bd1xx.py

```diff
@@ -217,7 +209,6 @@ def test_authors_from_100__a_e_w_y_and_700_a_e_w_y():
                     'value': 'B.A.Knyazev.2',
                 },
             ],
-            'curated_relation': False
```

Comment:
This one might be left a part. When it's False or the field is not there is basically the same thing.